dashboard/README.md

```diff
@@ -1,70 +1,198 @@
+    │   │       └── MainLayout
+    │   │           ├── index.jsx
+    │   │           └── index.less
+    │   ├── reducers
+    │   │   ├── authReducer.js
+    │   │   ├── datasetListReducer.js
+    │   │   ├── endpointReducer.js
+    │   │   ├── index.js
+    │   │   ├── loadingReducer.js
+    │   │   ├── navbarReducer.js
+    │   │   ├── overviewReducer.js
+    │   │   ├── tableOfContentReducer.js
+    │   │   ├── toastReducer.js
+    │   │   └── userProfileReducer.js
+    │   ├── store
+    │   │   └── store.js
+    │   ├── utils
+    │   │   ├── axiosInstance.js
+    │   │   ├── dateFunctions.js
+    │   │   ├── filterDataset.js
+    │   │   ├── helper.js
+    │   │   └── routeConstants.js
+    │   ├── App.css
+    │   ├── App.js
+    │   ├── App.test.js
+    │   ├── index.css
+    │   ├── index.js
+    │   ├── logo.svg
+    │   ├── reportWebVitals.js
+    │   └── setupTests.js
+    ├── config-overrides.js
+    ├── jsconfig.json
+    ├── jsdoc.json
+    ├── package.json
+    ├── package-lock.json
+    └── README.md
+```
+
+## Assets
+
+Assets placed in the `src/assets/images` directory are only referenced within component or layout definitions and are packaged in the generated `***.js` file during the build process.
+
+## Prerequisites
+
+## Install Node JS
+Refer to https://nodejs.org/en/ to install nodejs
+
+## Install create-react-app
+Install create-react-app npm package globally. This will help to easily run the project and also build the source files easily. Use the following command to install create-react-app
+
+```bash
+$ npm install -g create-react-app
+```
+
+## Cloning and Running the Application in local
+
+Clone the project into local
+Install all the npm packages. Go into the project folder and type the following command to install all npm packages
+
+```bash
+$ npm install
+```
+In order to run the application Type the following command
+
+```bash
+$ npm run dev
+```
+The Application Runs on localhost:3000
+
+## Build
+
+Build Application
+
+```bash
+$ npm run build
+```
+This will generate the `build` folder in the root directory, which contains packaged files such as `***.js`, `***.css`, and `index.html`.
+
+## Local Development
+
+Both the production and development builds of the dashboard require API endpoint configurations in order to query data from specific datastores.
+
+`server/index.js` contains references to datastores and metadata required to visualize data in the dashboard. 
+
+Create `.env` file in the root directory and delare the environment variable
+
+```bash
+$ PBENCH_SERVER=http://test_domain.com
+```
+
+## Storage
+
+Pbench Dashboard stores data using local browser storage and cookies.
+
+## Template
+
+This application is based on v1 of Ant Design Pro which is a production-ready UI solution for admin interfaces. For more information regarding the foundation and template of the application, please visit [https://v1.pro.ant.design/docs/getting-started](https://v1.pro.ant.design/docs/getting-started).
```

Comment:
Are we actually using any Ant Design? I thought it was all Patternfly?

Reply:
N.B. there's an Ant Design reference in line 191, too.

Reply:
Changed to PatternFly